terraform/modules/vault/variables.tf

```diff
@@ -58,6 +58,7 @@ variable "alb_allowed_ingress_cidrs" {
 ############################
 variable "alb_certificate_arn" {
   type        = "string"
+  default     = ""
```

Comment:
Very minor but convention for this module's variables is type, description, default in that order. Could we put this under the description.